client/overview/strings.tsx

```diff
@@ -26,7 +26,7 @@ export default {
 				mixedString: sprintf(
 					/* translators: %1$s: WooPayments */
 					__(
-						'{{bold}}%1s is in dev mode.{{bold /}}. You need to set up a live %1s account before you can accept real transactions.',
+						'{{bold}}%1s is in sandbox mode.{{bold /}}. You need to set up a live %1s account before you can accept real transactions.',
```

Comment:
It looks like there is an extra period after `{{bold /}}`.

Reply:
Oh! Good catch, Emily!

Reply:
@dmallory42 Could we do a tiny subsequent PR to fix what Emily shared?